docs/support_playbook.md

```diff
@@ -0,0 +1,59 @@
+Support Playbook
+================
+
+## Removing an organisation
+
+To remove an organisation, you can `discard` it:
+
+```ruby
+RecruitmentCycle.current.providers.find_by(provider_code: "1JJ").discard
+```
+
+If the org has running courses, you will get a validation error. In this scenario you just need to make sure there aren't any users attached to it.
+
+## Republishing a withdrawn course
+
+To republish a course which has been withdrawn:
+
+```ruby
+# Find the course by code or urn
+course = RecruitmentCycle.current.providers.find_by(urn: "138459").courses.find_by(course_code: "3C2F")
```

Comment:
probably best to user `provider_code` or `id` as `urn` has no unique constraint

Reply:
done